src/utils.ts

```diff
@@ -36,8 +36,8 @@ export const getToday = (minDate?: Date, maxDate?: Date): DateObj => {
   };
 };
 
-export const formatDate = (date: Date | null, dateFormat: string) =>
-  date ? format(startOfDay(date), convertTokens(dateFormat)) : undefined;
+export const formatDate = (date: Date | null, dateFormat: string, formatOptions: Object | null) =>
+  date ? formatOptions ? format(startOfDay(date), convertTokens(dateFormat), formatOptions): format(startOfDay(date), convertTokens(dateFormat)) : undefined;
```

Comment:
Please reuse the type created for the prop declaration and turn `formatOptions` into an optional param.

Once that's done, we don't need a new ternary for it, just pass it to the `format` function as the last param.

Reply:
Done